app/backend/src/clj/teet/integration/integration_email.clj

```diff
@@ -3,11 +3,10 @@
   (:require [cognitect.aws.client.api :as aws]
             [clojure.string :as str]
             [teet.environment :as environment]
+            [postal.core :as postal.core]
```

Comment:
no need to use `:as` if no alias is given, you can just require `postal.core` (without the brackets)